tests/foreman/cli/test_repository.py

```diff
@@ -908,6 +908,40 @@ def test_positive_synchronize_docker_repo(self, repo, module_product, module_org
             ]
         )
 
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
+        'repo_options',
+        **parametrized(
+            [
+                {
+                    'content-type': 'docker',
+                    'docker-upstream-name': CONTAINER_UPSTREAM_NAME,
+                    'name': valid_docker_repository_names()[0],
+                    'url': CONTAINER_REGISTRY_HUB,
+                }
+            ]
+        ),
+        indirect=True,
+    )
+    def test_verify_checksum_non_yum_repo(self, repo):
+        """Check if Verify Content Checksum can be run on non yum/deb repos
+
+        :id: c8f0eb45-3cb6-41b2-aad9-52ac847d7bf8
+
+        :parametrized: yes
+
+        :customerscenario: true
+
+        :expectedresults: Docker repository is created, and can be synced with
+            validate-contents set to True
+
+        :BZ: 2161209
+        """
+        assert repo['sync']['status'] == 'Not Synced'
+        Repository.synchronize({'id': repo['id'], 'validate-contents': 'true'})
+        new_repo = Repository.info({'id': repo['id']})
+        assert new_repo['sync']['status'] == 'Success'
```

Comment:
Do we've add any additional check that could be added to verify `'validate-contents': 'true'`, other than sync status?

Reply:
I can't think of a way. I think this is fine, as if it wasn't possible, you would get an error code returned from the command.